src/components/utility/UtilitySrv.js

```diff
@@ -0,0 +1,21 @@
+(function (angular) {
+    'use strict';
+
+    angular.module('znk.infra.utility').factory('UtilitySrv', [
+        function () {
+            var UtilitySrv = {};
+
+            UtilitySrv.general = {};
+
+            UtilitySrv.general.createGuid = function(){
```

Comment:
Oren says he has tested it with thousands generated ids and none was repeated :)